src/v2/coins/bch.js

```diff
@@ -45,33 +46,34 @@ class Bch extends AbstractUtxoCoin {
   /**
    * Canonicalize a Bitcoin Cash address for a specific version
    *
-   * Starting on January 14th, 2018 Bitcoin Cash's bitcoin-abc node switched over to using bech32
+   * Starting on January 14th, 2018 Bitcoin Cash's bitcoin-abc node switched over to using cashaddr
    * encoding for all of their addresses in order to distinguish them from Bitcoin Core's
```

Comment:
I think we need a period to end the sentence here, otherwise it makes it seem that Bitcoin Core's address format can be found at the link which follows.

Reply:
Done.